modules/pipeline/providers/dbgc/dbgc.go

```diff
@@ -42,13 +42,16 @@ const (
 // these two methods will create a lot of etcd ttl, will cause high load on etcd
 // use fixed gc time, traverse the data in the database every day
 func (p *provider) PipelineDatabaseGC(ctx context.Context) {
-	ticker := time.NewTicker(1 * time.Millisecond)
+	ticker := time.NewTicker(1 * time.Second)
 	defer ticker.Stop()
 	for {
 		select {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
+			// reset ticker to default for next gc
+			ticker.Reset(p.Cfg.PipelineDBGCDuration)
```

Comment:
issue occurred if dbgc duration is 1 millisecond.

use rutil.continuework

Reply:
adjusted